### DISCUSSION TITLE
Not able to set Name and color for Y Axes in LiveChart

### DISCUSSION THREAD

**Discussion by Emianna** [unoplatform/uno#18238] [General]
I am using LiveChartsCore.SkiaSharpView.WinUI for plotting a chart.  The "Name" and "LabelsPaint" properties for Y axis are not working for me. but those are working for X axis. Can you help me here? @Agnès Zitte @Beto Rodriguez @unoplatform

**Comment by jeromelaban** [comment]
Thanks for the report. This would be a question for the https://github.com/beto-rodriguez/LiveCharts2 repository.
